### DISCUSSION TITLE
Confused Liverness or Readness url      http://:metrics/healthz,  is it wrong?

### DISCUSSION THREAD

**Discussion by FumingD** [milvus-io/milvus#18601] [Q&A and General discussion] [answered]
we deploy milvus by the way of k8s cluster, and the milvus pod running normally.
But I found Liverness or Readness url   that http://:metrics/healthz,  a confused url without host。

**Comment by FumingD** [comment]
![微信图片_20220811123723](https://user-images.githubusercontent.com/35284373/184065639-4ce453ad-c951-4e1a-9ead-ff8c9edb11f4.jpg)

**Comment by LoveEachDay** [comment, marked as answer]
@FumingD This is just a short summary for liveness and readiness probe which means access the container's`metrics` port and `/healthz` as request path.

You can learn more from [here](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-http-request).
